src/components/Cluster/ClusterDetail/ClusterApps.js

```diff
@@ -142,6 +142,19 @@ const Disclaimer = styled.p`
   line-height: 1.2;
 `;
 
+const PreinstalledApps = styled.div`
+  display: flex;
+
+  & > div {
+    flex: 1;
+    margin-right: 15px;
+
+    &:last-child {
+      margin-right: 0px;
+    }
```

Comment:
Is it possible to rewrite these selectors to use classes to avoid a problem like you found in https://github.com/giantswarm/happa/pull/1617/files#r448168726

Reply:
Is it possible to rewrite these in the same sense as discussed in https://github.com/giantswarm/happa/pull/1617#discussion_r448390365?

I see this is not as severe as the duplicated components like https://github.com/giantswarm/happa/pull/1617/files#diff-d8b088a074d65e4a00a05a47c1687d4fL176 but would be nevertheless good to be resolved.

Giving an approve so this won't block you